launcher/image/test/test_basic.sh

```diff
@@ -0,0 +1,48 @@
+ #!/bin/bash
+set -euo pipefail
+source util/read_serial.sh
+
+# This test requires the workload to run and printing
+# corresponding messages to the serial console.
+SERIAL_OUTPUT=$(read_serial $1 $2) 
+print_serial=false
+
+if echo $SERIAL_OUTPUT | grep -q 'workload running'
+then
+    echo "- workload running verified"
+else
+    echo "FAILED: workload not running"
+    echo 'TEST FAILED.' > /workspace/status.txt
+    print_serial=true
+fi
+
+if echo $SERIAL_OUTPUT | grep -q 'workload args: \[/main arg_foo\]'
+then
+    echo "- arguments verified"
+else
+    echo "FAILED: arguments not verified"
+    echo 'TEST FAILED.' > /workspace/status.txt
+    print_serial=true
+fi
+
+if echo $SERIAL_OUTPUT | grep -q 'env_bar=val_bar'
```

Comment:
where are arg_foo and val_bar passed in?

Reply:
It is part of the image (when building with the Dockerfile), we can test cmd/env override in a separate test.